clients/iceberg-extensions/src/test/java/org/projectnessie/spark/extensions/ITNessieStatements.java

```diff
@@ -0,0 +1,365 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.spark.extensions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.projectnessie.client.tests.AbstractSparkTest;
+import org.projectnessie.error.BaseNessieClientServerException;
+import org.projectnessie.error.NessieConflictException;
+import org.projectnessie.error.NessieNotFoundException;
+import org.projectnessie.model.Branch;
+import org.projectnessie.model.CommitMeta;
+import org.projectnessie.model.ContentsKey;
+import org.projectnessie.model.IcebergTable;
+import org.projectnessie.model.ImmutableCommitMeta;
+import org.projectnessie.model.ImmutableOperations;
+import org.projectnessie.model.Operation;
+import org.projectnessie.model.Operations;
+import org.projectnessie.model.Tag;
+
+public class ITNessieStatements extends AbstractSparkTest {
+
+  private String hash;
+  private final String refName = "testBranch";
+
+  @BeforeAll
+  protected static void createDelta() {
+    conf.set(
+        "spark.sql.extensions", "org.projectnessie.spark.extensions.NessieSparkSessionExtensions");
+  }
+
+  @BeforeEach
+  void getHash() {
+    try {
+      hash = nessieClient.getTreeApi().getDefaultBranch().getHash();
+    } catch (NessieNotFoundException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  @AfterEach
+  void removeBranches() throws NessieConflictException, NessieNotFoundException {
+    for (String s : new String[] {refName, "main"}) {
```

Comment:
nit: `for (String s : Arrays.asList(refName, "main")) {`

Reply:
fixed